components/Theme/index.tsx

```diff
@@ -0,0 +1,11 @@
+import MoonIcon from './MoonIcon';
+
+const ThemeSwitch = () => {
+  return (
+    <div>
+      <MoonIcon />
+    </div>
```

Comment:
This is obviously a work in progress- but let's remember to add the cursor pointer back to the switch because I suggested removing it from the whole div.